src/devices/MAX31856/README.md

```diff
@@ -0,0 +1,51 @@
+﻿# MAX31856
+
+## Summary
+The MAX31856 device is a SPI interface cold-junction compensated thermocouple to digital converter.
+
+## Sensor Image
+![](device.jpg)
+
+## Usage
+The MAX31856.samples file contains a sample usage of the device. Note that this reads two temperatures. One is a connected thermocouple reading called ```temp``` and the other is the temperature of the device itself called ```tempCJ``` which is used internally to increase the accuracy of the thermocouple but can also be read if you find a use for it.
+
+Create a new ```SpiConnectionSettings``` Class if using a raspberry pi3 do not change these settings.
```

Comment:
`Raspberry Pi` (version doesn't matter) and then the cast is important as it's a brand.